### ISSUE TITLE
Double clicking a class opens a menu and then it get stuck open.

### ISSUE THREAD

**Issue by Place1** [josh-/MonashTimetable#2]
if you double click a class it opens the info thingy where you can change the colour. then if you click anywhere else it doesn't close. have to quit the application and reopen to fix it. (can only quick by opening activity monitor and killing process, perhaps add a quick button to the right click context menu when the users right clicks the icon)

**Comment by josh-** [comment]
Hmmm, does this consistently happen with every class that you view?

I haven't experience that, (the popover's behaviour is set to `NSPopoverBehaviorTransient` which is supposed to hide when the user interacts with any other UI element) so it may be an El Capitan regression. Can you reproduce this @dsykesturner?

**Comment by dsykesturner** [comment]
Yep I can reproduce it, however it only occurs with the first class colour your change. Then you can't interact with the popover afterwards so it may be just El Capitan

**Comment by Place1** [comment]
I did a fresh install to Yosemite today and when I tried to replicate the issue i could, so the issue still happens on Yosemite but in addition to the issue, a moment (1 maybe 2 seconds) after clicking somewhere else in the window the app crashed and closed. no error messages either.
